Update gherkin/step-organization.md

selfeki opened the pull request:
* Fix punctuation errors in the introduction and 'Step definition' section
* Rewrite sentence for clarity in 'Grouping steps' section
* Improve word choice in 'Helper methods' section

selfeki committed:
Update gherkin/step-organization.md

* Fix punctuation error in the introduction
* Rewrite sentence for clarity in 'Grouping steps' section
* Improve word choice in 'Helper methods' section

olleolleolle reviewed (approved):
The formatting is ace!

selfeki committed:
Update step-organization.md

Modify 'How Cucumber finds your features and step definitions' section for clarity
